lib/block-supports/elements.php

```diff
@@ -43,7 +43,7 @@ function gutenberg_render_elements_support( $block_content, $block ) {
 	}
 	$link_color_declaration = esc_html( safecss_filter_attr( "color: $link_color" ) );
 
-	$style = "<style>.$class_name a{" . $link_color_declaration . ";}</style>\n";
+	$style = ".$class_name a{" . $link_color_declaration . ';}';
```

Comment:
Nit pick: we can do $style = `".$class_name a{$link_color_declaration;}";`.

Reply:
Nah, that's a syntax error (having a `;` next to a variable `$link_color_declaration`). Note that the existing string concatenation also has it in separate strings for the same reason.